### PR TITLE
please merge new API

### PR DESCRIPTION
**Pull request by mwp007**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -24,7 +24,7 @@
   ],
   "repository": {
     "type": "git",
-    "url": "https://github.com/iobroker-community-adapters/ioBroker.samsung-community/"
+    "url": "https://github.com/mwp007/ioBroker.samsung-community/"
```
Comment (Apollon77) [inline package.json:27]: We should remove that change or ?! :-)

File: package.json [hunk 2]
```diff
@@ -42,7 +42,7 @@
     "chai": "^4.3.3"
   },
   "bugs": {
-    "url": "https://github.com/iobroker-community-adapters/ioBroker.samsung-community/issues"
+    "url": "https://github.com/mwp007/ioBroker.samsung-community/issues"
```
Comment (Apollon77) [inline package.json:45]: same here

**Review by Apollon77 — CHANGES_REQUESTED** [review]
Is the "pin" we now enter in settings only used once or regulary?

**Comment by Apollon77** [comment]
Please also rebase your PR ideally on master ... I updated deps and stuff

**Comment by mwp007** [comment]
The pin for the HJ Series is entered only once.

**Comment by Apollon77** [comment]
Cool Thx, so only the repo URLs are wrong - can you change that in your PR please?

**Comment by Apollon77** [comment]
(sorry wrong button)

File: package.json (hunk 1 above)
Comment (mwp007) [reply package.json:27]: oh zes, sure :-)

**Comment by GermanBluefox** [comment]
@jogibear9988 Can you check too?

**Comment by mwp007** [comment]
OK guys. I hope I did what you requested, but I was swimming in dark waters with the rebase. It*s rally a long time ago I coded last time.

File: package.json (hunk 1 above)
Comment (mwp007) [reply package.json]: I was following a tutorial, so sorry for that

**Comment by mwp007** [comment]
> 
> 
> Please also rebase your PR ideally on master ... I updated deps and stuff

I was kinda struggeling here

**Comment by Apollon77** [comment]
> I was kinda struggeling here

You must rebase your branch onto master ... there should be the IDE supporting you here :-)

If not possible then we can also merge and see status afterwards :-)

**Comment by mwp007** [comment]
> 
> 
> > I was kinda struggeling here
> 
> You must rebase your branch onto master ... there should be the IDE supporting you here :-)
> 
> If not possible then we can also merge and see status afterwards :-)

Zes, we should probablby do that. I was doing it with Visual Studio, but seems not to be succesfull. Need to read more.

**Comment by Apollon77** [comment]
We also need to update the issue templates because the ones from alexa2 were used

**Pull request merged**